bundles/org.eclipse.orion.client.ui/web/plugins/iframeCommunicationPlugin.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Should this be wrapped in <html> tags, or just a script tag?  I don't know how the plugin loading in orion works, but if this is just appended to the dom I would think it would just need to be wrapped in script tags?